### PR TITLE
user/update mockup

### PR DESCRIPTION
**Pull request by pkakelas**

### CODE REVIEW

**Comment by dionyziz** [comment]
Screenshot.

**Comment by pkakelas** [comment]
screenshot from 2014-05-19 14 32 07

Effect video: https://www.youtube.com/watch?v=1qomWh2mmNk

File: mockups/user/update.html [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+                        <a href="#" id="logout">Log out</a>
+                    </li>
+                    <form id="logout-form" action="session/delete" method="post" >
+                        <input type="hidden" name="token"/>
+                    </form>
+                </ul>
+            </div>
+        </div>
+        <div class="container" id="main">
+            <h1 class='text-center' id='title'>Personal Settings<h1>
```
Comment (vitsalis) [inline mockups/user/update.html]: maybe just `Settings` ?

File: mockups/user/update.html [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+        <div class="container" id="main">
+            <h1 class='text-center' id='title'>Personal Settings<h1>       
+                <form id='user-update-form' class="form-horizontal">
+                    <div class="form-group">
+                        <label class="col-sm-2 control-label">Username</label>
+                        <div class="col-sm-10">
+                            <input type="text" class="form-control" placeholder="pkakelas" disabled='true'/>
+                        </div>
+                    </div>
+                    <div class="form-group">
```
Comment (vitsalis) [inline mockups/user/update.html:115]: `name` and `surname`? We don't have these features.

File: mockups/user/update.html [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+                        <div class="col-sm-10" id='name-input'>
+                            <input type="text" class="form-control" placeholder="Name">
+                            <input type="text" class="form-control" placeholder="Surname">
+                        </div>
+                    </div>
+                    <div class="form-group form-inline">
+                        <label class="col-sm-2 control-label">Birthday</label>
+                        <div id='dob' class="row">
+                            <select class="form-control dob-input">
+                                <option>day</option>
```
Comment (vitsalis) [inline mockups/user/update.html:126]: you only have the `day` option. Will you fix that when you implement it?

File: mockups/user/update.html [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+                        <div class="col-sm-10">
+                            <select id='country-input' class="form-control">
+                                <option>Country</option>
+                            </select>
+                        </div>
+                    </div>
+                    <div class="form-group">
+                        <label class="col-sm-2 control-label">Email</label>
+                        <div class="col-sm-10">
+                            <input type="email" class="form-control" placeholder="Email">
```
Comment (vitsalis) [inline mockups/user/update.html]: value should be the user's current email

File: mockups/user/update.html [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+                        </div>
+                    </div>
+                    <div class="form-group">
+                        <label class="col-sm-2 control-label">Site</label>
+                        <div class="col-sm-10">
+                            <input type="email" class="form-control" placeholder="Site">
+                        </div>
+                    </div>
+                    <div class="form-group">
```
Comment (vitsalis) [inline mockups/user/update.html:156]: we do not have this feature yet. It should be commented out.

File: mockups/user/update.html [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+                        <label class="col-sm-2 control-label">Github</label>
+                        <div class="col-sm-10">
+                            <input type="email" class="form-control" placeholder="Github">
+                        </div>
+                    </div>
+                    <div class="form-group">
+                        <label for="Password3" class="col-sm-2 control-label">Password</label>
+                        <div class="col-sm-10" id='password-input'>
+                            <button  id='pswd-change' class="btn btn-default">Change password</button>
+                            <input type="password" class="form-control" placeholder="Old Password">
```
Comment (vitsalis) [inline mockups/user/update.html]: The user should have to put his password in order to change any of his account settings.

File: mockups/user/update.html (hunk 2 above)
Comment (pkakelas) [inline mockups/user/update.html:115]: Yes, I think we should add them. @dionyziz what do you think. They are going to be optional.

File: mockups/user/update.html (hunk 3 above)
Comment (pkakelas) [inline mockups/user/update.html:126]: Of cource

File: mockups/user/update.html (hunk 6 above)
Comment (pkakelas) [inline mockups/user/update.html]: Are you sure about this? I've never seen this.

File: mockups/user/update.html (hunk 2 above)
Comment (dionyziz) [inline mockups/user/update.html:115]: OK

File: mockups/user/update.html [hunk 7]
```diff
@@ -0,0 +1,208 @@
...
+			</p>
+			<p>
+			<button class='btn btn-default' id='close-modal'>Cancel</button>
+			<button class='btn btn-danger' id='delete-button'>Delete</button>
+			</p>
+		</aside>
+        <script>
+            $( '#delete-account' ).click( function() {
+                $( ".navbar" ).fadeTo( "slow",0.15 );
+				Avgrund.show( "#default-popup" );
```
Comment (dionyziz) [inline mockups/user/update.html]: nit: indent

File: mockups/user/update.html [hunk 8]
```diff
@@ -0,0 +1,208 @@
...
+                After the deletion of your End Of Codes account you will not be able to participate to the games.
+			</p>
+			<p>
+			<button class='btn btn-default' id='close-modal'>Cancel</button>
+			<button class='btn btn-danger' id='delete-button'>Delete</button>
+			</p>
+		</aside>
+        <script>
+            $( '#delete-account' ).click( function() {
+                $( ".navbar" ).fadeTo( "slow",0.15 );
```
Comment (dionyziz) [inline mockups/user/update.html:194]: nit: space after `,`

File: mockups/user/update.html [hunk 9]
```diff
@@ -0,0 +1,208 @@
...
+				Avgrund.show( "#default-popup" );
+            } );
+            $( '#close-modal' ).click( function() {
+                $( ".navbar" ).fadeTo( "slow", 1 );
+				Avgrund.hide( "default-popup" );
```
Comment (dionyziz) [inline mockups/user/update.html]: nit: indent

File: mockups/user/update.html (hunk 8 above)
Comment (dionyziz) [inline mockups/user/update.html:194]: Perhaps fade to `0`?

Comment (pkakelas) [inline mockups/user/update.html:194]: I tried it the way you suggest and I find it more beautiful the way it is now. Do you want video with you suggestion?

**Pull request merged**